src/microbe_stage/ProcessSystem.cs

```diff
@@ -43,6 +43,15 @@ public ProcessSystem(Node worldRoot)
         return result;
     }
 
+    /// <summary>
+    ///   Computes the energy balance for the given organelles in biome
+    /// </summary>
+    public static EnergyBalanceInfo ComputeEnergyBalance(IEnumerable<OrganelleTemplate> organelles,
+        BiomeConditions biome, MembraneType membrane)
+    {
+        return ComputeEnergyBalance(organelles.Select(i => i.Definition), biome, membrane);
```

Comment:
You used `organelles.Select(organelle => organelle.Definition)` in `MicrobeEditor.cs`, so I'd advocate for consistency in one way or another.

Reply:
I've converted everything doing that select to use this overload instead. Good catch.